app/src/main/java/org/mozilla/fenix/tabstray/ext/TabSessionState.kt

```diff
@@ -13,3 +13,9 @@ fun TabSessionState.filterFromConfig(configuration: Configuration): Boolean {
 
     return content.private == isPrivate
 }
+
+fun TabSessionState.getTrayPosition(): Int =
+    when (content.private) {
+        true -> 1
+        false -> 0
+    }
```

Comment:
nit: If you use `Page` enum values, that might be just one source of truth for these values. Adds a bit more symmetry with the `positionToPage` too. 😄 

```suggestion
    when (content.private) {
        true -> Page.PrivateTabs.ordinal
        false -> Page.NormalTabs.ordinal
    }
```

Reply:
Good idea, I'll update